### PR TITLE
Xenoarcheaology removal

### PR DESCRIPTION
**Pull request by sgtmind**
## About The Pull Request

The removal of xenoarchaeology and all its components in order to make way for something better, or at least with far less spaghetti code. (I know nothing about why it is spaghetti)

## Why It's Good For The Game

Honestly, had an awful experience with a random artifact spawning once, and I realized that xenoarcheaology adds nothing to the game. In addition, it is a bunch of spaghetti code

## Changelog
:cl:
del: Removed xenoarcheology 
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
my one request is to not edit the mapping of XenoArch in eris itself as i have plans for it and dont want to go through pipe hell

**Comment by MLGTASTICa** [comment]
absolutely useless PR thta only serves to remove content for no reason and to replace it with absolutely nothing.
Xenoarch doesn't lag and does have some uses , people don't choose to use it and thats fine , but we shouldn't be removing content that doesn't create major balance issues or performance

**Comment by ValoTheValo** [comment]
> absolutely useless PR thta only serves to remove content for no reason and to replace it with absolutely nothing. Xenoarch doesn't lag and does have some uses , people don't choose to use it and thats fine , but we shouldn't be removing content that doesn't create major balance issues or performance

Xeno Arch is nearly unused in the code. its not used for anything. it breaks the lore. its ancient bay Jank. and there isn't really anything worth using left in it

**Review by Firefox13 — COMMENTED** [review]
Xenoarch is very old code (even if it works) and usually goes unused. I don't see any reason not to remove it.

File: cev_eris.dme [hunk 1]
```diff
@@ -671,7 +663,6 @@
 #include "code\game\machinery\floodlight.dm"
 #include "code\game\machinery\floor_light.dm"
 #include "code\game\machinery\floorlayer.dm"
-#include "code\game\machinery\gym.dm"
```
Comment (ValoTheValo) [inline cev_eris.dme:674]: i think your branch is old. gym isnt a part of xenoarch

**Review by SirRichardFrancis — CHANGES_REQUESTED** [review]
Based PR. That system is ancient and works poorly, half the time someone decides to use it - server pays in performance. Remember that artifact that tried to recharge every cell in the world every few seconds? Or the one that spawned invisible OP gun? The ones that spawned morbillion simple mobs maybe? I do remember and I want none of it to repeat.

Also, if we want a feature like that - we'll make one. Something like "One Star prototypes" idk. Certainly not alien artifacts, there is no aliens in ~Reereverse~ Eris lore.

This PR should be remade on up-to-date branch though.

**Comment by sgtmind** [comment]
Right, will take a bit for me to fully get it back up to current spec.

**Comment by sgtmind** [comment]
I only closed this so I can redo the thing from scratch, attempt to get on an up to date branch.

**Pull request closed without merging**